spec/30-processing-model.md

```diff
@@ -29,7 +29,7 @@ forwards Trace Context headers.
    `parent-id`.
        - If either `trace-id`, `parent-id` or `trace-flags` are invalid, the tracer
       creates a new `traceparent` header and deletes `tracestate`.
-5. The tracer validates the `tracestate` header. If the `tracestate` header
+5. The tracer _MAY_ validate the `tracestate` header. If the `tracestate` header
```

Comment:
"If the `tracestate` header cannot be parsed the tracer deletes it."

-> should we also change this to "If the `tracestate` header cannot be parsed the tracer, _MAY_ discard the whole header. It _MAY_ as well discard only the invalid entries."

Reply:
I updated the comment with your suggestion with some slight edits. Let me know if it needs any further clarification or wordsmithing.